products/compute/api.yaml

```diff
@@ -11316,6 +11316,61 @@ objects:
           PARTNER InterconnectAttachment is created, updated,
           or deleted.
         output: true
+      - !ruby/object:Api::Type::Array
```

Comment:
This should be a NestedObject instead of an array of NestedObjects since bfd is an object in the schema (https://cloud.google.com/compute/docs/reference/rest/beta/routers#Router.FIELDS.inlinedField_18). That should fix the panic you're seeing :)

Reply:
Stupid me, completely missed this. Should be good now.